src/python/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+find_package(Python COMPONENTS Interpreter Development)
+find_package(pybind11 CONFIG)
```

Comment:
no version checks ?

Reply:
I don't know what minimum version we need tbh

Reply:
hum, which version did you test with ? Ubuntu 20/debian 9 ships with 2.0.1

Reply:
The last pybind11 release. I can try with 2.0.1

Reply:
The first compatible version is 2.2, I've updated the code.